indexer-app/src/cmr/indexer/data/concepts/collection/opendata.clj

```diff
@@ -56,3 +56,15 @@
   [publication-reference]
   (when-let [doi (-> publication-reference :DOI :DOI)]
     (doi->url doi)))
+
+(defn collection-citation->opendata-citation
+  "Returns subset of collection-citation needed for opendata response."
+  [collection-citation]
+  (let [citation-keys [:Creator
```

Comment:
Since there is no variable in `citation-keys`, use `def` to define `citation-keys` and remove the `let` block.